src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -172,6 +175,11 @@ const Footer = BaseFooter.extend({
 });
 
 export default BaseView.extend({
+  Header: BaseHeader.extend({
```

Comment:
Can you use `BaseAuthenticatorView.js` which already defines beacon header.

Reply:
The API response for this view is not for a factor, using BaseAuthenticatorView won't generate the right class for beacon.